openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -1942,7 +1942,6 @@ def _assert_existing_user_error(self, response):
         for conflict_attribute in ["username", "email"]:
             self.assertIn(conflict_attribute, errors)
             self.assertIn("belongs to an existing account", errors[conflict_attribute][0]["user_message"])
-        self.assertNotIn("partial_pipeline", self.client.session)
```

Comment:
See above for all these changes in this file. If it makes it easier, it may be OK to just check for the new `partial_pipeline_token` key.